__tests__/app.test.js

```diff
@@ -92,3 +92,88 @@ describe('GET /api/articles/:article_id', () => {
         })
     })
 })
+
+describe('GET /api/articles', () => {
+    test('status:200, responds with a JSON object containing an array of article objects, each of which contains the following properties: author, title, article_id, topic, created_at, votes, article_img_url', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+
+        const { articles } = body;
+
+        expect(articles).toHaveLength(5);
+
+        articles.forEach((article) => {
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String)
+                    })
+        })
+        })
+    })
+    test('status:200, each article object has a comment_count property which is the total count of all comments with that article_id', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+            
+            const { articles } = body;
+
+            articles.forEach((article) => {
+                expect(article).toHaveProperty("comment_count", expect.any(String));
```

Comment:
We could pop this in the previous test's forEach, just to make things a bit neater. 👍